src/blog/v8-release-85.md

```diff
@@ -0,0 +1,143 @@
+
+## JavaScript
+
+### `Promise.any` and `AggregateError`
+
+`Promise.any` is a promise combinator that resolves the resulting promise as soon as one of the input promises is fulfilled.
+
+```js
+const promises = [
+  fetch('/endpoint-a').then(() => 'a'),
+  fetch('/endpoint-b').then(() => 'b'),
+  fetch('/endpoint-c').then(() => 'c'),
+];
+try {
+  const first = await Promise.any(promises);
+  // Any of the promises was fulfilled.
+  console.log(first);
+  // → e.g. 'b'
+} catch (error) {
+  // All of the promises were rejected.
+  console.assert(error instanceof AggregateError);
+  // Log the rejection values:
+  console.log(error.errors);
+}
+```
+
+If all input promises are rejected, the resulting promise is rejected with an `AggregateError` object containing an `errors` property which holds an array of rejection values.
+
+Please see our explainer for more.
+
+### `String.prototype.replaceAll`
+
+`String.prototype.replaceAll` provides an easy way to replace all occurrences of a substring without creating a global `RegExp`.
+
+```js
+const queryString = 'q=query+string+parameters';
+
+// Works, but requires escaping inside regular expressions.
+queryString.replace(/\+/g, ' ');
+// → 'q=query string parameters'
+
+// Simpler!
+queryString.replaceAll('+', ' ');
+// → 'q=query string parameters'
+```
+
+Please see our explainer for more.
+
+### Logical assignment operators
+
+Logical assignment operators are new compound assignment operators that combine the logical operations `&&`, `||`, or `??` with assignment.
+
+```javascript
+x &&= y;
+// Roughly equivalent to x && (x = y)
+x ||= y;
+// Roughly equivalent to x || (x = y)
+x ??= y;
+// Roughly equivalent to x ?? (x = y)
+```
+
+Note that, unlike mathematical and bitwise compound assignment operators, logical assignment operators only conditionally perform the assignment. Please read our explainer for a more in-depth explanation.
+
+## WebAssembly
+
+### Liftoff shipped on all platforms
+
+Since v6.9, [Liftoff](https://v8.dev/blog/liftoff) has been used as the baseline compiler for WebAssembly on Intel platforms (Chrome enabled it on desktop systems since M-69). Since we were concerned about memory increase (because of more code being generated by the baseline compiler), we held it back for mobile systems so far. After some experimentation in the last months, we are confident that the memory increase is negligible for most cases, hence we finally enable Liftoff by default on all architectures, bringing increased compilation speed, especially on arm devices (32- and 64-bit). Chrome follows along and ships Liftoff in M-85.
+Multi-value support shipped
```

Comment:
Should be a header. Are you testing + previewing this locally? Doing so might make it easier to spot these kinds of things